x/axelarnet/keeper/msg_server.go

```diff
@@ -356,7 +356,7 @@ func (s msgServer) RegisterAsset(c context.Context, req *types.RegisterAssetRequ
 
 	// also register on axelarnet, it routes assets from cosmos chains to evm chains
 	// ignore the error in case above chain is axelarnet, or if the asset is already registered
-	_ = s.nexus.RegisterAsset(ctx, exported.Axelarnet, nexus.NewAsset(req.Asset.Denom, false), req.Limit, req.Window)
+	_ = s.nexus.RegisterAsset(ctx, exported.Axelarnet, nexus.NewAsset(req.Asset.Denom, false), utils.MaxUint, types.DefaultRateLimitWindow)
```

Comment:
I'd prefer to remove this additional axelar registration entirely, but it'll break some e2e tests/scripts